terraform/variables.tf

```diff
@@ -115,3 +115,25 @@ variable "sgtm_feature__check_rerun_on_approval_enabled" {
   description = "'true' if a check rerun should be rerequested when a PR is approved. Must be combined with sgtm_feature__check_rerun_base_ref_names and sgtm_feature__check_rerun_threshold_hours."
   default     = "false"
 }
+
+variable "github_app_name" {
+  type        = string
+  default     = null
+  description = "The name of the Github app that will be used to authenticate with Github."
+}
+
+variable "github_app_installation_access_token_retrieval_url" {
+  type        = string
+  default     = null
+  description = "The URL to retrieve a Github app installation access token from. This URL should accept a POST request with a JSON body that specifies a value for the 'github_app_name' key. The URL should return a JSON object with a 'token' key that contains the Github token, and an 'expires_at' key which contains a timestamp of the format '%Y-%m-%dT%H:%M:%SZ'."
+}
+
+variable "custom_config_bucket_name" {
+  type        = string
+  description = "The S3 bucket that stores custom configuration for your deployment of SGTM"
+}
+variable "custom_lambda_role_policy_s3_object_key" {
```

Comment:
Why are these in both the `terraform/variables.tf` and `terraform/sgtm-cluster/vars.tf` ?

Reply:
this is an unfortunate terraform limitation - think of 'sgtm-cluster` as a class we create 2 instances of -- `prod` and `staging`. `terraform/sgtm-cluster/vars.tf` defines the 'initialization parameters' for the `sgtm-cluster` class. 

But, our terraform workspace also needs many of the same variables to be defined -- it needs those var values so that it can pass them to `sgtm-cluster` when it instantiates it. 

Terraform doesn't have a nice way to say 'the initialization args for this workspace are the same as the module' or vice versa, so we end up needing to repeat a lot of the variable definitions twice :(